src/adapters/refract/transformResource.coffee

```diff
@@ -43,6 +43,7 @@ module.exports = (resourceElement, location, options) ->
     attributes = _.dataStructures(transitionElement)
```

Comment:
Okay, I am confused. This PR works correctly when there are `resource` attributes and `action` attributes. So, I am wondering what is this line's use at all.